.github/workflows/test.yml

```diff
@@ -146,12 +146,13 @@ jobs:
 
       - name: Test paths to Haskell backend binaries
         run: |
-          nix develop github:runtimeverification/k/sam/nix-kore-integration-shell#kore-integration-tests \
+          K_VERSION=v$(cat deps/k_release)
```

Comment:
doing `v$(cat deps/k_release)` twice. can we remove `K_VERSION=v$(cat deps/k_release)`?

Reply:
I might also remove the path checking - that was more for debugging while it did not work properly yet